src/publisher/rempl-publisher.ts

```diff
@@ -11,6 +11,9 @@ import {
   RecordEventHandler,
   Message,
 } from "./types";
+import { hook } from "./index";
```

Comment:
Publisher should be abstract from hook, overlay, etc. It should just call some provided methods. There an integration via `RemoteCommandsApi` were added, see https://github.com/lahmatiy/react-render-tracker/blob/main/src/publisher/rempl-publisher.ts#L136

Reply:
Fixed